tests/test_taskipy.py

```diff
@@ -179,6 +177,35 @@ def test_running_task_arguments_not_passed_to_post_hook(self):
         self.assertEqual(exit_code, 0)
 
 
+class ListTaskTestCase(TaskipyTestCase):
+
+    def test_running_task_list(self):
+        cwd = self.create_test_dir_from_fixture('project_with_tasks_to_list')
+        exit_code, stdout, _ = self.run_task('--list', cwd=cwd)
+
+        expected = "\n".join([
+            "one    echo first task",
+            "two    echo second task",
+            "three  echo third task",
+        ])
+        self.maxDiff = None
+        self.assertEqual(expected, stdout.strip())
+        self.assertEqual(exit_code, 0)
+
+    def test_running_task_list_with_arg(self):
+        cwd = self.create_test_dir_from_fixture('project_with_tasks_to_list')
+        exit_code, stdout, _ = self.run_task('does_not_exist', ['--list'], cwd=cwd)
```

Comment:
As per my comment, in this test we **want** to trigger the task (and not the list directive). Therefore, let's change this line to one of the existing commands:
```suggestion
        exit_code, stdout, _ = self.run_task('one', ['--list'], cwd=cwd)
```
and expect this command to run.